pandas/core/common.py

```diff
@@ -405,16 +407,27 @@ def random_state(state=None):
     Returns
     -------
     np.random.RandomState
+
+    ..versionchanged:: 1.1.0
```

Comment:
this needs to be in the Paramters section, just below the 'If receives anything'